torch/utils/_pytree.py

```diff
@@ -323,24 +335,22 @@ def register_constant(cls: type[Any]) -> None:
 
     """
 
-    def _flatten(x):  # type: ignore[no-untyped-def]
-        return [], ConstantNode(x)
-
-    def _unflatten(_, context):  # type: ignore[no-untyped-def]
-        return context.value
-
-    def _flatten_with_keys(x):  # type: ignore[no-untyped-def]
-        return [], ConstantNode(x)
-
     _private_register_pytree_node(
         cls,
-        _flatten,
-        _unflatten,
-        flatten_with_keys_fn=_flatten_with_keys,
+        _constant_flatten,
+        _constant_unflatten,
+        flatten_with_keys_fn=_constant_flatten_with_keys,
     )
 
 
-@dataclasses.dataclass
+def is_constant_cls(cls: type[Any]) -> bool:
```

Comment:
nit suggestion: `is_constant_class`

```suggestion
def is_constant_class(cls: type[Any]) -> bool:
```

Reply:
Updated.